### PR TITLE
multi-org support if query follow <org>/<query> pattern

### LINKED ISSUE #17: support multiple chef orgs
authentication to the chef server just needs one key and then permission to the orgs it requests searches from. simple implementation allowing support for multiple orgs:
requests to the app follow `/<org>/<query>`

examples:
- `GET /org-a/tags:some-tag`
- `GET /org-b/tags:some-tag`

each of those requests would hit their respective chef servers

to allow backwards compatibility, the chef org could be a get parameter.

Possibly in the future this could even support multiple chef servers.

### PR DESCRIPTION
**Pull request by jiokmiso**
Hi @atheiman 
I think this might fix #17 

Tagging @jp017129

### CODE REVIEW

File: lib/better_chef_rundeck.rb [hunk 1]
```diff
@@ -102,8 +116,12 @@ def get_filter_result params_hsh
 
     # format nodes for yaml: {<name>: {<attr>: <value>, <attr>: <value>}}
     formatted_nodes = {}
+
+    # processing query with organization(<organizatio>/<query>)
+    chef_server_url, serach_query = filter_organization(query)
+
     # query the chef server
-    Chef::Search::Query.new.search(:node, query, filter_result: filter_result) do |node|
+    Chef::Search::Query.new(chef_server_url).search(:node, serach_query, filter_result: filter_result) do |node|
```
Comment (JP017129) [inline lib/better_chef_rundeck.rb]: Typo here on serach_query

**Comment by JP017129** [comment]
+1

**Comment by JP017129** [comment]
@atheiman, would you be able to review and merge this please?

**Comment by atheiman** [comment]
there needs to be rspec unit tests for this new functionality.

**Comment by atheiman** [comment]
Also would be good to see a test for /main/<query> response == /<query> response

**Comment by jiokmiso** [comment]
I removed the ridley, because it seems like it cannot handle the multi-org queries. Now it uses only chefzero for its testing. Let me know if this change undesirable. And now it has different test data, and does have /main == / test.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ resources.source.1.config.url=http\://better-chef-rundeck.example.com/role:webse
 
 The returned data includes Chef attributes that will be attached to nodes in Rundeck, allowing users to search for nodes in Rundeck based on Chef attributes.
 
+```
+GET /organization/role:webserver
+```
+
+will return the same result as above method for specified organization.
```
Comment (atheiman) [inline README.md]: there needs to be more information in the readme about multi org support than this.

**Comment by jiokmiso** [comment]
I tried to be more verbose on the readme. I am not super good at those, so I am open to suggestion as well. Or you can just overwrite mine.

**Pull request merged**